### PR TITLE
Updating README with guidance to use rootless Docker for linux users

### PR DESCRIPTION
**Pull request by aro5000**
I noticed on the installation page in vscode that linux users are recommended to enable the Docker CLI for non-root users, but this can have some significant security considerations. I am proposing to add a section before this that recommends a rootless Docker configuration. I am running Ubuntu Desktop 20.04 and rootless Docker appears to work well with the extension.

### CODE REVIEW

**Review by karolz-ms — APPROVED** [review]
Thank you @aro5000 , makes sense

**Review by bwateratmsft — APPROVED** [review]
I'm lacking a Linux VM to try this out with but I'll trust @aro5000 that this works :)

**Comment by ucheNkadiCode** [comment]
I approve this after changing (most secure) to (more secure)

**Comment by aro5000** [comment]
@ucheNkadiCode it looks like only those with write access to the repo can merge. Is there anything else I can provide for this? :)

**Comment by bwateratmsft** [comment]
Merging it. Thanks a ton @aro5000!

**Pull request merged**
